webrtc/legacy/RTCPeerConnection-createOffer-offerToReceive.html

```diff
@@ -0,0 +1,274 @@
+<!doctype html>
```

Comment:
Can you add a README file to the legacy/ directory saying that these tests exercise code from section 4.4.3 "Legacy extensions", APIs that are optional to implement?

Reply:
OK